src/robot/libdoc.py

```diff
@@ -64,9 +64,16 @@
 Options
 =======
 
- -f --format HTML|XML     Specifies whether to generate HTML or XML output.
+ -f --format HTML|XML|XML:HTML
+                          Specifies whether to generate HTML or XML output.
                           If this options is not used, the format is got
                           from the extension of the output file.
+                          Embedding HTML documentation in XML output:
+                          If XML should have embedded HTML in <doc> as
+                          documentation, it is possible to configure 
+                          this with 'XML:HTML'. This could be used if the XML
+                          files are used to be imported into other systems
+                          that require HTML text to present this to a human.
```

Comment:
Same comments about docs I wrote related to the User Guide documentation apply also to this help text. The help text doesn't need to be as thorough as User Guide, though.

Reply:
I think it is not disturbing the user, when it is very briefly explained here.
often Manpages are too lean. Imho.

i would let it here because in the moment it has no negative effect.

Reply:
In my opinion it should be explicitly documented in the body of the help text. Usage should be short.